__tests__/PaginationBoxView-test.js

```diff
@@ -1474,4 +1474,198 @@ describe('Test custom props', () => {
+          .getAttribute('rel')
+      ).toBe('custom-prev-rel');
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:nth-child(4) a')
+          .getAttribute('rel')
+      ).toBe('custom-next-rel');
+    });
+    it('should not render prevPageRel and nextPageRel if pageCount is 1', function () {
+      const pagination = ReactTestUtils.renderIntoDocument(
+        <PaginationBoxView pageCount={1} />
+      );
+
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:first-child a')
+          .getAttribute('aria-label')
+      ).toBe('Previous page');
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:nth-Child(2) a')
+          .getAttribute('rel')
+      ).toBe('canonical');
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('.selected a')
+          .getAttribute('rel')
+      ).toBe('canonical');
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:nth-last-Child(2) a')
+          .getAttribute('rel')
+      ).toBe('canonical');
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:last-child a')
+          .getAttribute('aria-label')
+      ).toBe('Next page');
+    });
+    it('should not render prevPageRel if selected page is first', function () {
+      const pagination = ReactTestUtils.renderIntoDocument(
+        <PaginationBoxView pageCount={4} />
+      );
+
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:nth-Child(1) a')
+          .getAttribute('aria-label')
+      ).toBe('Previous page');
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:nth-Child(2) a')
+          .getAttribute('rel')
+      ).toBe('canonical');
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:nth-Child(3) a')
+          .getAttribute('rel')
+      ).toBe('next');
+    });
+    it('should not render nextPageRel if selected page is last', function () {
+      const pagination = ReactTestUtils.renderIntoDocument(
+        <PaginationBoxView pageCount={4} />
+      );
+
+      const activeItem = ReactDOM.findDOMNode(pagination).querySelector(
+        'li:nth-last-child(2) a'
+      );
+      ReactTestUtils.Simulate.click(activeItem);
+
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:nth-last-Child(1) a')
+          .getAttribute('aria-label')
+      ).toBe('Next page');
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:nth-last-Child(2) a')
+          .getAttribute('rel')
+      ).toBe('canonical');
+      expect(
+        ReactDOM.findDOMNode(pagination)
+          .querySelector('li:nth-last-Child(3) a')
+          .getAttribute('rel')
+      ).toBe('prev');
+    });
+    it('should not render prevPageRel if the break page is present just before the selected page', function () {
```

Comment:
Aren't these 2 tests description inverted?

On this one it says _should **not** render **prevPageRel**_, but then we find the `prev` rel.